PRIVACY.md

```diff
@@ -34,10 +34,11 @@ The Name Suggestion Index is a feature in iD to aid users in consistently and ac
 
 ### Contact
 
-If you believe that we hold information that would allow us to correct, amend, or delete inaccurate information about you or if you believe that information about you has been processed in violation of this privacy policy or applicable law, please email us at privacy@ideditor.org.
+If you believe that we hold information that would allow us to correct, amend, or delete inaccurate information about you or if you believe that information about you has been processed in violation of this privacy policy or applicable law, please email us at [legal-questions@osmfoundation.org](mailto:legal-questions@osmfoundation.org).
```

Comment:
IMHO, you need to check with the LWG, but at least historically privacy@osmfoundation,org has been the contact address for privacy related matters.